plugins/obs-filters/data/locale/en-US.ini

```diff
@@ -3,6 +3,7 @@ ColorGradeFilter="Apply LUT"
 MaskFilter="Image Mask/Blend"
 AsyncDelayFilter="Video Delay (Async)"
 CropFilter="Crop/Pad"
+HdrTonemapFilter="HDR Tonemap"
```

Comment:
Just on the wording here, could it be made more obvious that using this filter is optional (perhaps "HDR Tonemap Override")?
I’m a bit worried that otherwise users could be under the impression (perhaps influenced by bad YouTube tutorials) that they need this for every HDR source if their OBS is set to SDR.

Reply:
I was thinking Override too.

Reply:
I went with "HDR Tonemap (Override)"

Reply:
wonderful, thanks